flexmeasures/api/v3_0/sensors.py

```diff
@@ -712,6 +724,24 @@ def patch(self, sensor_data: dict, id: int, sensor: Sensor):
         :status 403: INVALID_SENDER
         :status 422: UNPROCESSABLE_ENTITY
         """
+        audit_log_data = list()
+        for k, v in sensor_data.items():
+            if getattr(sensor, k) == v:
```

Comment:
Clearer if it is just `getattr(sensor, k) != v:` then we don't need `continue`, right?